frontend/src/utilities/useWatchComponents.tsx

```diff
@@ -15,7 +15,7 @@ export const useWatchComponents = (
   const initForce = React.useRef<number>(forceUpdate);
 
   React.useEffect(() => {
-    let watchHandle: ReturnType<typeof setTimeout>;
+    let watchHandle: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Remove the if condition in the return statement.
```suggestion
    let watchHandle: ReturnType<typeof setTimeout>;
```